src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ManagedIndexMetaData.kt

```diff
@@ -44,6 +44,7 @@ data class ManagedIndexMetaData(
     val actionMetaData: ActionMetaData?,
     val stepMetaData: StepMetaData?,
     val policyRetryInfo: PolicyRetryInfoMetaData?,
+    val errorNotificationFailure: String?,
```

Comment:
What is the reason that this error message is outside of the info? Can't we add it as part of existing info map and  add one more entry?

Reply:
Was thinking info was mainly for step/action related info whereas errorNotificationFailure is a failure regarding their configuration of the error_notification at the root level of the policy. But, it looks like we do dump other runner/policy related info in there, so will move it inside info too.